x-pack/plugins/maps/public/shared/layers/sources/es_join_source.js

```diff
@@ -180,4 +185,12 @@ export class ESJoinSource extends AbstractESSource {
   async filterAndFormatPropertiesToHtml(properties) {
     return await this.filterAndFormatPropertiesToHtmlForMetricFields(properties);
   }
+
+  async createESTooltipProperty(propertyName, rawValue) {
+    const indexPattern = await this._getIndexPattern();
```

Comment:
This could throw an exception if the index pattern does not exist. Need to handle exception case and return null when index pattern not found